tools/base/envoy_python.bzl

```diff
@@ -315,10 +315,11 @@ def envoy_py_data(name, src, format = None, entry_point = base_entry_point):
     native.genrule(
         name = name_env,
         cmd = """
-        PICKLE_PATH=$$(realpath %s) \
+        PICKLE_DATA=$$(cat %s | base64) \
         && echo -n "\
-               \nfrom envoy.base.utils.data_env import DataEnvironment \
-               \ndata = DataEnvironment.load(\\"$$PICKLE_PATH\\")" \
+               \nimport base64 \
+               \nimport pickle \
+               \ndata = pickle.loads(base64.b64decode(\\"\\"\\"$$PICKLE_DATA\\"\\"\\"))\n" \
```

Comment:
it might be possible to remove the b64 en/decoding - i was struggling to get the pickle into a string

not sure why the `$(location x)` s are not working in these macros - there is a similar problem with the jinja macro - in that case it needs a separate file so the only way i managed to get it working was by using the fact that the data file is a sibling of the lib the macro creates

Reply:
i have tried removing the b64 - but cant seem to make it work - there is not a great cost decoding b64 tho, so not a big issue